stl/inc/format

```diff
@@ -3519,7 +3519,10 @@ struct _Format_checker {
     consteval explicit _Format_checker(basic_string_view<_CharT> _Fmt) noexcept
         : _Parse_context(_Fmt, _Num_args), _Parse_funcs{&_Compile_time_parse_format_specs<_Args, _ParseContext>...} {}
     constexpr void _On_text(const _CharT*, const _CharT*) const noexcept {}
-    constexpr void _On_replacement_field(size_t, const _CharT*) const noexcept {}
+    constexpr void _On_replacement_field(const size_t _Id, const _CharT*) const {
+        _ParseContext _Parse_ctx({});
+        _Parse_ctx.advance_to(_Parse_funcs[_Id](_Parse_ctx));

```

Comment:
Why bother updating the state of this parse context that we're about to drop on the floor?
```suggestion
        (void) _Parse_funcs[_Id](_Parse_ctx);
```